### PR TITLE
feat(adopters): add cncf and linux foundation usage details

### PR DESCRIPTION
**Pull request by kmova**
Include the usage details provided under the comment:
https://github.com/openebs/openebs/issues/2719#issuecomment-595105716

Signed-off-by: kmova <kiran.mova@mayadata.io>

### CODE REVIEW

File: adopters/cncf/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+- nginx (serving home pages, backups, CSV reports etc)
+- Storage for git repositories clones
+- All DevStats applications (homegrown)
+
+The Linux Foundation:
+- MariaDB database (Helm master + slave replication, automatic backups)
+- ElasticSearch cluster (Helm chart master + 4 slave nodes)
+- Postgres (including automatic backups)
+- Redis
+- Grimoire stack (CHAOSS)
```
Comment (ksatchit) [inline adopters/cncf/README.md]: should we provide one more link here or below: https://chaoss.community/

Comment (lukaszgryglicki) [reply adopters/cncf/README.md]: Those are for the Linux Foundation, list for CNCF is different. My comment had CNCF and LF cases separated.

Comment (ksatchit) [reply adopters/cncf/README.md]: Ok, thanks @lukaszgryglicki

**Comment by ksatchit** [comment]
LGTM

**Pull request merged**